config.py.example

```diff
@@ -0,0 +1,2 @@
+class BotConfig:
+    token = "1234567890:ABCD_abcdefghijklmnopqrstuvwxyz1234"
```

Comment:
token should be written in capitals if I read the code correctly.. 
And perhaps an example of the DbConfig can be added? (DbConfig.SQLALCHEMY_DATABASE_URI)

Reply:
I added the DbConfig class to the config.py